### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by onkar75** [AmruthPillai/Reactive-Resume#1265]
**Describe the bug**

Is it possible to have multipage resume ? Could not find any way to use it , I have more content so how does it can be created in 2-3 page resume.

**Product Flavor**

- [ ] Managed (https://rxresu.me)
- [ ] Self Hosted

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

- OS: 
- Browser 
- Version 

**Additional context**

**Comment by AMDphreak** [comment]
I agree this is necessary but you have it labeled as a bug. Change the label to Feature Request, if you can, and give your post a title. [bug] is not descriptive.

**Comment by AMDphreak** [comment]
Strangely, you have to manually add new pages to the resume in the builder. It doesn't automatically handle the overflow.

![image](https://github.com/user-attachments/assets/1361f5b3-1893-4eb8-b432-9158ca33aabe)

**Comment by AMDphreak** [comment]
duplicate of #1834

**Comment by AmruthPillai** [comment]
This issue was automatically closed as it was created before 2024-01-01.
